02_bubble/brnbke/main_test.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("[1 2 3 5]\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main(), expected %v, sadly got %v", expected, actual)
+	}
+}
+
+var testCases = map[string]struct {
+	input []int
```

Comment:
Given that these two pieces of the struct are of the same type this could be simplified to a single line

Reply:
Yes, you're right, did not notice that, though not sure what the expected way of doing this is in go TBH.